consensus/misc/create2deployer.go

```diff
@@ -0,0 +1,31 @@
+package misc
+
+import (
+	"github.com/ethereum-optimism/superchain-registry/superchain"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+var create2DeployerAddress = common.HexToAddress("0x13b0D85CcB8bf860b6b79AF3029fCA081AE9beF2")
+var create2DeployerCodeHash = common.HexToHash("0xb0550b5b431e30d38000efb7107aaa0ade03d48a7198a140edda9d27134468b2")
+var create2DeployerCode []byte
+
+func init() {
+	code, err := superchain.LoadContractBytecode(superchain.Hash(create2DeployerCodeHash))
```

Comment:
https://github.com/ethereum-optimism/superchain-registry/pull/14 would need to be merged first

Reply:
PR is merged